examples/kubernetes/lib/custom_filter.py

```diff
@@ -0,0 +1,4 @@
+import base64
```

Comment:
can we rename the file and function to `custom_jinja2_filter` ? so that it's clearer for people new to kapitan. :)